mechanics/tools/gen-aliases/main.go

```diff
@@ -63,8 +63,13 @@ func main() {
 
 `, infile)
 	output = append([]byte(preamble), output...)
-	ioutil.WriteFile(outfile, output, 0644)
-	log.Print("Wrote ", outfile)
+	prevOut, err := ioutil.ReadFile(outfile)
```

Comment:
As a follow up can you drop the use of `ioutil.ReadFile` it's deprecated

https://pkg.go.dev/io/ioutil#ReadFile

I guess we don't lint the code in this repo

Reply:
Opened an issue for the linting here https://github.com/knative/community/issues/1276.

@Bisht13 Do you mind updating the `ReadFile` request above or at least open an issue if you don't have the time. That was my bad for not catching that sorry.

Reply:
I'll fix that

Reply:
@dprotaso made an issue #1278 and opened a PR #1279 ✨